crescent/plugin.py

```diff
@@ -251,7 +266,16 @@ def _load(self, client: Client) -> None:
         for callback in self._load_hooks:
             callback()
         for child in self._children:
-            add_hooks(client, child)
+            if isinstance(child.metadata, AppCommandMeta):
+                child.metadata.add_hooks(client.command_hooks, after=False)
+                child.metadata.add_hooks(client.command_after_hooks, after=True)
+            if isinstance(child.metadata, EventMeta):
+                child.metadata.add_hooks(  # pyright: ignore [reportUnknownMemberType]
+                    client.event_hooks, after=False
+                )
+                child.metadata.add_hooks(  # pyright: ignore [reportUnknownMemberType]
+                    client.event_after_hooks, after=True
+                )
```

Comment:
think the same thing needs to be done here? Can we just use the `add_hooks` function now?